corehq/apps/registration/forms.py

```diff
@@ -490,23 +491,24 @@ class AdminInvitesUserForm(SelectUserLocationForm):
 
     def __init__(self, data=None, excluded_emails=None, is_add_user=None,
                  role_choices=(), should_show_location=False, can_edit_tableau_config=False,
-                 *, domain, **kwargs):
+                 custom_data=None, *, domain, **kwargs):
+        self.custom_data = custom_data if WEB_USER_INVITE_ADDITIONAL_FIELDS.enabled(domain) else None
+        if data and self.custom_data:
+            data = data.copy()
+            custom_data_post_dict = self.custom_data.form.data
+            data.update({k: v for k, v in custom_data_post_dict.items() if k not in data})
         self.request = kwargs.get('request')
         super(AdminInvitesUserForm, self).__init__(domain=domain, data=data, **kwargs)
         self.can_edit_tableau_config = can_edit_tableau_config
         domain_obj = Domain.get_by_name(domain)
         self.fields['role'].choices = [('', _("Select a role"))] + role_choices
         if domain_obj:
-            if domain_has_privilege(domain_obj.name, privileges.APP_USER_PROFILES):
```

Comment:
Might be missing something but why is this FF not being used anymore? Is it effectively putting all this behind the new flag?

Reply:
It should still be behind the domain privilege check, but it's now done from the `CustomDataEditor` form [here](https://github.com/dimagi/commcare-hq/blob/0698b3dfcde9554bb4107545b0e9fe9d8de943e1/corehq/apps/custom_data_fields/edit_entity.py#L147) If that conditional is `false`, then the `profile` field [here](https://github.com/dimagi/commcare-hq/blob/0698b3dfcde9554bb4107545b0e9fe9d8de943e1/corehq/apps/custom_data_fields/edit_entity.py#L170) won't be added to the fields meaning that `self.custom_data.form.fields` will also not contain the profile field.